src/pypromice/process/value_clipping.py

```diff
@@ -29,31 +29,34 @@ def clip_values(
 
     variable_limits = var_configurations[cols].dropna(how="all")
     for var, row in variable_limits.iterrows():
-        if var not in list(ds.variables):
-            continue
-
-        if var in ["rh_u_cor", "rh_l_cor"]:
-            ds[var] = ds[var].where(ds[var] >= row.lo, other=0)
-            ds[var] = ds[var].where(ds[var] <= row.hi, other=100)
-
-            # Mask out invalid corrections based on uncorrected var
-            var_uncor = var.split("_cor")[0]
-            ds[var] = ds[var].where(~np.isnan(ds[var_uncor]), other=np.nan)
-
-        else:
-            if ~np.isnan(row.lo):
-                ds[var] = ds[var].where(ds[var] >= row.lo)
-            if ~np.isnan(row.hi):
-                ds[var] = ds[var].where(ds[var] <= row.hi)
-
-        other_vars = row.OOL
-        if isinstance(other_vars, str) and ~ds[var].isnull().all():
-            for o in other_vars.split():
-                if o not in list(ds.variables):
-                    continue
-                else:
-                    if ~np.isnan(row.lo):
-                        ds[var] = ds[var].where(ds[var] >= row.lo)
-                    if ~np.isnan(row.hi):
-                        ds[var] = ds[var].where(ds[var] <= row.hi)
+        try:
+            if var not in list(ds.variables):
+                continue
+    
+            if var in ["rh_u_cor", "rh_l_cor"]:
+                ds[var] = ds[var].where(ds[var] >= row.lo, other=0)
+                ds[var] = ds[var].where(ds[var] <= row.hi, other=100)
+    
+                # Mask out invalid corrections based on uncorrected var
+                var_uncor = var.split("_cor")[0]
+                ds[var] = ds[var].where(~np.isnan(ds[var_uncor]), other=np.nan)
+    
+            else:
+                if ~np.isnan(row.lo):
+                    ds[var] = ds[var].where(ds[var] >= row.lo)
+                if ~np.isnan(row.hi):
+                    ds[var] = ds[var].where(ds[var] <= row.hi)
+    
+            other_vars = row.OOL
+            if isinstance(other_vars, str) and ~ds[var].isnull().all():
+                for o in other_vars.split():
+                    if o not in list(ds.variables):
+                        continue
+                    else:
+                        if ~np.isnan(row.lo):
+                            ds[var] = ds[var].where(ds[var] >= row.lo)
+                        if ~np.isnan(row.hi):
+                            ds[var] = ds[var].where(ds[var] <= row.hi)
+        except:
+            pass
```

Comment:
Sorry, but what string is incompatible and what kind of exception is raised?
It is dangerous to just catch all exception without at least logging it :-)

You could do something like:

```python
except TheExpectedExceptionType as e:
    logger.exception(e)
```

And instantiate a local logger after the imports:

```python
logger = logging.getLogger(__name__)
```

Reply:
This is maninly because the GPS information is a `str` at this point, so I needed to write a work around that stopped it trying to clip string values.

Reply:
I've taken this out of the most recent changes, as I have come to a different solution with the suggestions from @BaptisteVandecrux